<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelicalLineWidget</class>
 <widget class="QWidget" name="HelicalLineWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Helical line widget</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="main_gbox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Line</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>2</number>
          </property>
          <item row="2" column="0">
           <widget class="QPushButton" name="create_line_button">
            <property name="text">
             <string>Create</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="overlay_slider">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>120</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="overlay_cbox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Display overlay</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="swap_points_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Swap points</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="4">
           <widget class="QTreeWidget" name="lines_treewidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>120</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>120</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Start point</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>End point</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="remove_line_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="create_auto_line_button">
            <property name="text">
             <string>Create auto line</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>overlay_cbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>overlay_slider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
